conference.js

```diff
@@ -242,6 +255,8 @@ function hangup (requestFeedback = false) {
     .catch(errCallback.bind(null, disconnect))
     .then(maybeRedirectToWelcomePage)
     .catch(errCallback.bind(null, maybeRedirectToWelcomePage));
+
+    _hangupInProgress = false;
```

Comment:
Shouldn't we move this line to a place where the Promise is finished? Maybe before maybeRedirectToWelcomePage method will be a good place. WDYT?